logging/bin/remove_fluentbit_k8sevents_opensearch.sh

```diff
@@ -3,15 +3,18 @@
 # Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cd "$(dirname $BASH_SOURCE)/../.."
+cd "$(dirname "$BASH_SOURCE")/../.." || {
+  echo "Failed to change directory"
+  exit 1
+}
```

Comment:
Why is this being handled differently in different scripts?  Here is echos a message when failing to change directory, in other places it doesn't.  Here it keeps the $BASH_SOURCE reference but in some places (e.g. logging/bin/remove_fluentbit_azmonitor.sh) it changes the reference to "$0"